<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Aerius Calculator Input</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_layer">
         <property name="text">
          <string>Input layer</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QgsMapLayerComboBox" name="combo_layer"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_sector">
         <property name="text">
          <string>Sector</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="combo_sector"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_subsector">
         <property name="text">
          <string>Subsector</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="combo_subsector"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_crs">
         <property name="text">
          <string>CRS</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="combo_crs"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab_metadata">
          <attribute name="title">
           <string>Metadata</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QGroupBox" name="group_project">
             <property name="title">
              <string>Project</string>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <item row="1" column="0">
               <widget class="QLabel" name="label_project_year">
                <property name="text">
                 <string>Year</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="combo_project_year"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_project_description">
                <property name="text">
                 <string>Description</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QTextEdit" name="edit_project_description">
                <property name="acceptRichText">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="group_situation">
             <property name="title">
              <string>Situation</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QFormLayout" name="formLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="label_situation_name">
                <property name="text">
                 <string>Name</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="edit_situation_name"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_situation_type">
                <property name="text">
                 <string>Type</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="combo_situation_type"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="group_calculation">
             <property name="title">
              <string>Calculation</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QFormLayout" name="formLayout_4">
              <item row="2" column="0">
               <widget class="QLabel" name="label_calculation_result_type">
                <property name="text">
                 <string>Result type</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="combo_calculation_result_type"/>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_calculation_type">
                <property name="text">
                 <string>Type</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="combo_calculation_type"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_calculation_substances">
                <property name="text">
                 <string>Substances</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QCheckBox" name="check_calculation_substance_nh3">
                  <property name="text">
                   <string>NH3</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="check_calculation_substance_nox">
                  <property name="text">
                   <string>NOx</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_emission_sources">
          <attribute name="title">
           <string>Emission Sources</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>468</width>
                <height>538</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="1" column="1">
                <widget class="QgsFieldComboBox" name="fcb_sector_id"/>
               </item>
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_label">
                   <property name="text">
                    <string>label</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_sector_id">
                 <property name="text">
                  <string>sectorId</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0" colspan="2">
                <widget class="QGroupBox" name="groupBox_es_characteristics">
                 <property name="title">
                  <string>Characteristics</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                 <layout class="QFormLayout" name="formLayout_5">
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_diurnal_variation">
                    <property name="text">
                     <string>diurnalVariation</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QgsFieldComboBox" name="fcb_diurnal_variation"/>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_emission_height">
                    <property name="text">
                     <string>emissionHeight</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QgsFieldComboBox" name="fcb_emission_height"/>
                  </item>
                  <item row="3" column="0" colspan="2">
                   <widget class="QGroupBox" name="groupBox_building">
                    <property name="title">
                     <string>Building</string>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                    <layout class="QFormLayout" name="formLayout_7">
                     <item row="0" column="0">
                      <widget class="QLabel" name="label_b_length">
                       <property name="text">
                        <string>length</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="0">
                      <widget class="QLabel" name="label_b_width">
                       <property name="text">
                        <string>width</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="0">
                      <widget class="QLabel" name="label_b_height">
                       <property name="text">
                        <string>height</string>
                       </property>
                      </widget>
                     </item>
                     <item row="3" column="0">
                      <widget class="QLabel" name="label_b_orientation">
                       <property name="text">
                        <string>orientation</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QgsFieldComboBox" name="fcb_b_length"/>
                     </item>
                     <item row="1" column="1">
                      <widget class="QgsFieldComboBox" name="fcb_b_width"/>
                     </item>
                     <item row="2" column="1">
                      <widget class="QgsFieldComboBox" name="fcb_b_height"/>
                     </item>
                     <item row="3" column="1">
                      <widget class="QgsFieldComboBox" name="fcb_b_orientation"/>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item row="2" column="0" colspan="2">
                   <widget class="QGroupBox" name="groupBox_heatContent">
                    <property name="title">
                     <string>heatContent</string>
                    </property>
                    <layout class="QFormLayout" name="formLayout_8">
                     <item row="0" column="0">
                      <widget class="QLabel" name="label_hc_value">
                       <property name="text">
                        <string>value</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QgsFieldComboBox" name="mFieldComboBox"/>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QgsFieldComboBox" name="fcb_label"/>
               </item>
               <item row="3" column="0" colspan="2">
                <widget class="QGroupBox" name="groupBox_emissions">
                 <property name="title">
                  <string>Emissions</string>
                 </property>
                 <layout class="QFormLayout" name="formLayout_6">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_em_nox">
                    <property name="text">
                     <string>NOX</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QgsFieldComboBox" name="fcb_em_nox"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_em_nh3">
                    <property name="text">
                     <string>NH3</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QgsFieldComboBox" name="fcb_em_nh3"/>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_road_transportation">
          <attribute name="title">
           <string>Roads</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_fn">
       <item>
        <widget class="QLabel" name="label_outfile">
         <property name="text">
          <string>Save as</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edit_outfile"/>
       </item>
       <item>
        <widget class="QPushButton" name="button_outfile">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
